APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
+| Site-to-cloud VPN | China Telecom, Huawei | 2023/07/23| [Link]([https://github.com/camaraproject/WorkingGroups/pull/273](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_Site%20to%20cloud%20(S2C)%20VPN.md)) | China Telecom, China Mobile, Huawei | Not approved (2023/09/20, 2023/10/19)| N/A | N/A | N/A |  N/A |  N/A |  N/A |  N/A |N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Device Swap-->%%%%%%%%%%%%%%%%%%%%%
+| Device Swap | MTN | 2023/08/13| [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/Device%20Swap.md) | GSMA-OGW, MTN, Huawei | Approved (2023/10/19)  | N/A | N/A | N/A | N/A | N/A | N/A | N/A |N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---IMEI Fraud-->%%%%%%%%%%%%%%%%%%%%%
+| IMEI Fraud | MTN | 2023/08/13| [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/IMEI%20Fraud.md) | GSMA-OGW, MTN, Huawei | Approved (2023/10/19)  | [DeviceIdentifier](https://github.com/camaraproject/DeviceIdentifier) | N/A| N/A | N/A | N/A | N/A | N/A |N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---WebRTC-->%%%%%%%%%%%%%%%%%%%%%
+| WebRTC | Telefonica | 2023/08/16 | [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_WebRTC_Telefonica.md) | GSMA-OGW, Mavenir, Orange, Telefonica, TMUS, Vodafone | Approved (2023/09/07) | [WebRTC](https://github.com/camaraproject/WebRTC) | sim_swap | [Link](https://github.com/camaraproject/WebRTC/tree/main/documentation) | N/A | N/A | N/A | N/A | N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Network Insights-->%%%%%%%%%%%%%%%%%%%%%
+| Network Insights | Verizon | 2023/08/31| [Link](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_NetworkInsights_Verizon.md) | 5GFF, GSMA-OGW, Verizon | Approved (2023/10/19) | N/A | N/A | N/A | N/A | N/A | N/A | N/A| N/A|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---Network Slicing-->%%%%%%%%%%%%%%%%%%%%%
+| Network Slicing | China Unicom | 2023/09/26| [Link](https://github.com/camaraproject/WorkingGroups/pull/316) |China Unicom | Under evaluation (YYYY/MM/DD)| N/A | N/A | N/A | N/A | N/A | N/A | N/A|N/A |%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%<!---KYC Age verification-->%%%%%%%%%%%%%%%%%%%%%
```

Comment:
https://github.com/camaraproject/WorkingGroups/pull/316 is closed.
Replacement seems to be https://github.com/camaraproject/WorkingGroups/pull/333